perf/benchmark/runner/runner.py

```diff
@@ -187,6 +187,7 @@ def run(self, conn, qps, size, duration):
                 labels=labels)
 
         if self.run_ingress:
+            print('--------------Running ingress mode--------------')
```

Comment:
Instead of `Running ingress mode`, could we rename it to be `Running in ingress mode`?

Reply:
done.